seeding/dataGeneration/generateListingData.js

```diff
@@ -0,0 +1,52 @@
+const faker = require('faker');
+const path = require('path');
+const fs = require('fs');
+
+const numNewRecords = 10000000;
+const batchSize = 100000;
+const dataFilePath = path.join(__dirname, 'listingDataWithIds.csv');
+
+const headers = [
+  'id',
+  'address',
+  'price_per_night',
+  'star_rating',
+  'min_stay',
+  'cleaning_fee',
+  'max_guests',
+  'state',
+  'country',
+];
+
+fs.writeFileSync(dataFilePath, `${headers.join(',')}\n`, 'utf-8');
```

Comment:
I know you used sync and it works, but I would recommend going back and learning how to use the streams they are pretty handy.

Reply:
what would be the benefit in this case?